test/parallel/test-dotenv-invalid-syntax.js

```diff
@@ -0,0 +1,69 @@
+// Flags: --env-file test/fixtures/dotenv/invalid-syntax.env
```

Comment:
We already have a `test-dotenv-edge-cases.ts` file. Can you move your test to there, please?